test-bench-utils/lib/sinks/nosqlInjection/rethinkdb/r-insert.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { r, dbInit, connectionParams } = require('./dbInitiation');
+
+/**
+ * @param {Object} params
+ * @param {string} params.input user input string
+ * @param {Object} opts
+ * @param {boolean} [opts.safe] are we calling the sink safely?
+ * @param {boolean} [opts.noop] are we calling the sink as a noop?
+ */
+module.exports = function reDbQuery(
+  { input },
+  { safe = false, noop = false } = {}
+) {
+  const result = new Promise((resolve, reject) => {
+    dbInit
+      .then(() => {
+        r.connect(connectionParams).then((conn) => {
+          if (noop) {
+            resolve('NOOP');
+          }
+          if (safe) {
+            resolve('For now no safe option possible');
```

Comment:
insert expects array of objects. I suppose that the input can be safe if some schema validation is enforced. We don't support it yet, that's right, but maybe it's a bit misleading to return a message that says that there is no safe option possible at all.